packages/paste-core/components/heading/src/index.tsx

```diff
@@ -3,63 +3,57 @@ import * as PropTypes from 'prop-types';
 import {Text} from '@twilio-paste/text';
 
 export type asTags = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'div' | 'label' | 'span';
-export type HeadingStyle =
-  | 'headingStyle10'
-  | 'headingStyle20'
-  | 'headingStyle30'
-  | 'headingStyle40'
-  | 'headingStyle50'
-  | 'headingStyle60';
+export type HeadingVariants = 'heading60' | 'heading50' | 'heading40' | 'heading30' | 'heading20' | 'heading10';
 
 interface Heading {
   as: asTags;
   id?: string;
   className?: never;
-  variant?: HeadingStyle;
+  variant?: HeadingVariants;
 }
 
-function getHeadingStyles(headingStyle?: HeadingStyle): {} {
-  switch (headingStyle) {
-    case 'headingStyle60':
+function getHeadingProps(headingVariant?: HeadingVariants): {} {
+  switch (headingVariant) {
+    case 'heading60':
```

Comment:
Isn't this supposed to be heading10?  Or are we leaving it inverted?

Reply:
I decided to leave it inverted because it goes against the rest of the scaling we use in token. I know it's not a token, but it just felt weird being opposite those. I'm not 100% tied to that idea, so open to changing.

Reply:
More words: I think as a user, component APIs !== design system core tokens / aliases.  I want the simplest component API.  As it is currently, I have to do `6 - hlevel` every time I want the size and the tag to match, which isn't intuitive.

Reply:
Yeah. I think that makes the most sense. I'll change real quick.

Reply:
Pushed those changes.